common.yaml

```diff
@@ -39,6 +39,9 @@ conditional-include:
   - if: osversion == "centos-10"
     include:
       - fedora-coreos-config/manifests/shared-el10.yaml
+  - if: osversion == "rhel-10.1"
```

Comment:
Interesting, this might be a dumb question; how often do overrides differ between centos10 (or 9) and any z stream?

Reply:
It all depends on the point in time :)

I'll open a PR after this one to illustrate this.

Reply:
As promised: https://github.com/coreos/rhel-coreos-config/pull/9